internal/sourcecode/scan.go

```diff
@@ -114,6 +115,44 @@ func configureDockerfile(sourceDir string) (*SourceInfo, error) {
 	return s, nil
 }
 
+func configureLucky(sourceDir string) (*SourceInfo, error) {
+	if !checksPass(sourceDir, dirContains("shard.yml", "lucky")) {
+		return nil, nil
+	}
+
+	s := &SourceInfo{
+		Family:     "Lucky",
+		Files:      templates("templates/lucky"),
+		Port:       8080,
+		ReleaseCmd: "lucky db.migrate",
+		Env: map[string]string{
+			"PORT":       "8080",
+			"LUCKY_ENV":  "production",
+			"APP_DOMAIN": "APP_FQDN",
+		},
+		Secrets: []Secret{
+			{
+				Key:      "SECRET_KEY_BASE",
+				Help:     "Lucky needs a random, secret key. Use the random default we've generated, or generate your own.",
+				Generate: true,
+			},
+			{
+				Key:   "SEND_GRID_KEY",
+				Help:  "Lucky needs a SendGrid API key. For now, we're setting this to unused. You can generate one at https://docs.sendgrid.com/for-developers/sending-email/api-getting-started",
+				Value: "unused",
+			},
+		},
+		Statics: []Static{
```

Comment:
This is only desirable if we don't care about cache headers. If we want that, and they are configurable in Lucky, then it's better to let Lucky serve the assets.

Reply:
I've asked the community, but I don't think these are configurable, so statics should be better.
Will there ever be the option to set cache headers on statics?

Reply:
We don't have specific plans now for the statics feature right now, so couldn't say. Most likely yes, or something more like a CDN will emerge.